### ISSUE TITLE
Error: template: test:85:33: executing "test" at <.Vars.machine_uuid>: map has no entry for key "machine_uuid"

### ISSUE THREAD

**Issue by m3freak** [ansible-lockdown/UBUNTU22-CIS-Audit#8]
This is my first time trying to use the goss binary to do an audit run. When I run it, I get the error in the subject. The command I'm using is:

```
goss --vars UBUNTU22-CIS-Audit-cis_release/vars/CIS.yml -g UBUNTU22-CIS-Audit-cis_release/goss.yml validate
```

The test for the `machine_uuid` that sets the `host_machine_uuid` variable runs just fine in the terminal.

Am I doing something incorrectly?

**Comment by uk-bolly** [comment]
hi @m3freak 

Apologies for the delay in response on this one. The goss.yml CIS file is looking for some facts that we set in the run_audit.sh script.
This is also how the ansible playbook runs the audit with the template created by the ansible rules and options selected and runs that same script allowing it to ensure it matches the desired state.

You are able to run goss as expected by just commenting out those in the goss.yml. The CIS.yml will also need manually adjusting to match your expected results.
A bit further information on this can be found in 

https://ansible-lockdown.readthedocs.io/en/latest/

I can see we have chatted on discord, feel free to ping on there if you wish.


thanks

uk-bolly
